src/main/java/org/mvplugins/multiverse/core/teleportation/TeleportQueue.java

```diff
@@ -12,7 +12,7 @@ public class TeleportQueue {
 
     private final Map<String, String> teleportQueue;
 
-    public TeleportQueue() {
+    TeleportQueue() {
         teleportQueue = new HashMap<>();
```

Comment:
Does it matter that we're not using a thread safe Map implementation here?

Reply:
thenAccept for completable future is not on an async thread right?